dask/dataframe/core.py

```diff
@@ -5202,7 +5206,10 @@ def cov_corr_agg(data, cols, min_periods=2, corr=False, scalar=False):
     with np.errstate(invalid="ignore", divide="ignore"):
         mat = C / den
     if scalar:
-        return mat[0, 1]
+        result = mat[0, 1]
+        if hasattr(result, "get"):
```

Comment:
Alternatively, we can skip this check and leave it up to dask_cudf to wrap the `corr` method and convert to scalar. What do you think @mrocklin ?

Reply:
Is this always a scalar?  If so I wonder if we could call something like `result = float(result)` or some other protocol that was likely to work in both cases.